<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monitoring</class>
 <widget class="QWidget" name="Monitoring">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>  PRINTER MANAGEMENT PROGRAM FOR AUTOMATED GATE SYSTEM</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icon/hutchisonport_logo.ico</normaloff>../Icon/hutchisonport_logo.ico</iconset>
  </property>
  <widget class="QLabel" name="lbPaperStatus">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>80</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PaperEnd Status :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPrinterStatus">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Printer Status :</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>1281</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_3">
    <widget class="QTableWidget" name="tableLogPrimary">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>651</width>
       <height>271</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>130</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>No.</string>
      </property>
     </row>
    </widget>
    <widget class="QPushButton" name="btnLogFilePrimary">
     <property name="geometry">
      <rect>
       <x>1200</x>
       <y>450</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #006CFF;
color:white;</string>
     </property>
     <property name="text">
      <string>LogFile</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QTableWidget" name="tableLogSecondary">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>651</width>
       <height>271</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>130</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>No</string>
      </property>
     </row>
    </widget>
    <widget class="QPushButton" name="btnLogFileSecondary">
     <property name="geometry">
      <rect>
       <x>1200</x>
       <y>450</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #006CFF;
color:white;</string>
     </property>
     <property name="text">
      <string>LogFile</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="lbLocation">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Location :</string>
   </property>
  </widget>
  <widget class="QWidget" name="bgPrinterManagementMonitoring" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #006CFF;
</string>
   </property>
   <widget class="QToolButton" name="btnmenu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Icon/MenuIcon.ico</normaloff>../Icon/MenuIcon.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="popupMode">
     <enum>QToolButton::InstantPopup</enum>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="PrinterStatus">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowSecondaryTable">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Secondary</string>
   </property>
  </widget>
  <widget class="QLabel" name="PaperEndStatus">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>80</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowPrimaryTable">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Primary</string>
   </property>
  </widget>
  <widget class="QLabel" name="Location">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbPrinterManagementMonitoring_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>1191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string> PRINTER MANAGEMENT - MONITORING</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPrinterModel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Printer Model :</string>
   </property>
  </widget>
  <widget class="QLabel" name="PrinterModel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbOnlineStatus">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Online Status :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPaperJamStatus">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>140</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PaperJam Status :</string>
   </property>
  </widget>
  <widget class="QLabel" name="OnlineStatus">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>140</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PaperJamStatus">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>140</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_information">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>1281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>lbPaperStatus</zorder>
  <zorder>lbPrinterStatus</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>lbLocation</zorder>
  <zorder>bgPrinterManagementMonitoring</zorder>
  <zorder>PrinterStatus</zorder>
  <zorder>PaperEndStatus</zorder>
  <zorder>btnShowPrimaryTable</zorder>
  <zorder>Location</zorder>
  <zorder>lbPrinterManagementMonitoring_2</zorder>
  <zorder>btnShowSecondaryTable</zorder>
  <zorder>lbPrinterModel</zorder>
  <zorder>PrinterModel</zorder>
  <zorder>lbOnlineStatus</zorder>
  <zorder>lbPaperJamStatus</zorder>
  <zorder>OnlineStatus</zorder>
  <zorder>PaperJamStatus</zorder>
  <zorder>lb_information</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
